src/test/coins_tests.cpp

```diff
@@ -562,20 +564,57 @@ BOOST_AUTO_TEST_CASE(ccoins_serialization)
     }
 }
 
+struct CoinEntry {
+    enum class State { CLEAN, DIRTY, FRESH, DIRTY_FRESH };
+
+    const CAmount value;
+    const State state;
+
+    CoinEntry(CAmount v, State s) : value(v), state(s) {}
```

Comment:
```suggestion
    constexpr CoinEntry(CAmount v, State s) : value(v), state(s) {}
```
then all `const static MaybeCoin` declarations below can be made `constexpr`.

Reply:
Done, thanks